sdk/python/kfp/v2/dsl/artifact.py

```diff
@@ -126,7 +127,7 @@ def __init__(self, instance_schema: Optional[str] = None):
       if 'properties' not in schema_yaml:
         raise ValueError('Invalid instance_schema, properties must be present. '
                          'Got %s' % instance_schema)
-      schema = schema_yaml['properties']
+      schema = schema_yaml['properties'] or {}
       self.TYPE_NAME = yaml.safe_load(instance_schema)['title']
```

Comment:
nit: just noticed this, should we check for `if 'title' not in schema_yaml:` as well?

Reply:
Done.